### DISCUSSION TITLE
Test case graphs not showing today data

### DISCUSSION THREAD

**Discussion by cesarsanabrialoft** [allure-framework/allure2#2094] [TestOps]
I created graph which type is Launch Trend (By date) and it is filter based on test cases. This graph shows data from yesterday August 28, when I have tests that ran today August 29.  Could you please tell me what is the delay time to show the data?
The launch shows the data came in. The launch is also close.

![image](https://github.com/allure-framework/allure2/assets/91267057/2fba275d-c3db-479d-8de6-664bc2fd87fc)

**Comment by cheshi-mantu** [comment]
hello @cesarsanabrialoft,
have you closed the launches? Data in statistics appear only after a launch is closed.

**Comment by PavelMal** [comment]
@cheshi-mantu 
good day!
Can confirm that the same behavior is reproducible for me too
Closed launch with one test:
<img width="1659" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/6f4edb41-07f7-4492-9219-73f0a39f1e6a">

No data is displayed for the 1st of the September
<img width="1292" alt="image" src="https://github.com/allure-framework/allure2/assets/12068504/abc0e6fe-e6ce-4463-9cd9-7eda7c1c2716">

**Reply by cheshi-mantu** [reply]
@PavelMal 
hi Pavel, 
could you please register a request at https://help.qameta.io, we need more details on your environment and time zones you reside.

cheers
